qiskit_nature/circuit/library/initial_states/fermionic_gaussian_state.py

```diff
@@ -141,6 +141,8 @@ def __init__(
         Raises:
             ValueError: transformation_matrix must be a 2-dimensional array.
             ValueError: transformation_matrix must have shape (n_orbitals, 2 * n_orbitals).
+            ValueError: transformation_matrix does not describe a valid transformation
+                of fermionic ladder operators
```

Comment:
I see the ValueError that gets raised does explain why its not considered valid. Which is presumably it fails to meet the specials constraints as referenced by the parameter docstring. This is the first part of the message in the ValueError but I wonder here if it said it does not meet the constraints required of the matrix it would tie more into the docstring of the parameter which notes this need and refers to the detail

I will note that when sphinx builds the docs the class and constructor docstring are merged to become one. So this original text `This matrix must satisfy special constraints; see the main body of the docstring of this class.` might be a little confusing when looking at the resultant html. It might be better/easier to simply say something like `This matrix must satisfy the constraints as detailed above.` (You can see the html if you unzip the documentation.zip asset that is created by the build where the assets are listed and can be downloaded via build Summary

Reply:
I've updated the docstring. I think the reference to the class docstring looks good in the html and is clearer than your suggestion. Let me know if you'd like me to change it.

Reply:
I don;t know this is how it comes out where the class docstring and constructor docstring are merged such that the args end up right below what you are referring to - and saying see main body of the docstring of this class, while correct, I am not sure that readers of the html necessarily see it that way - after all if you look there its right above where they need to look.

![image](https://user-images.githubusercontent.com/40241007/171638250-4a89758b-ebf9-489f-834d-99c7261635a2.png)

Reply:
Ok, I've taken your suggestion.